api/app/src/command/webhook/webhook.ts

```diff
@@ -122,6 +128,15 @@ export const processData = async <T extends MetriportData>(
   }
 };
 
+const reportUsage = async (cxId: string, userIds: string[]): Promise<void> => {
+  const users = await getConnectedUsers({ cxId, ids: userIds });
+  users.forEach(({ cxUserId }) => [
+    reportUsageCmd({ cxId, cxUserId }).catch((err) => {
+      log(`Failed to report usage, cxId ${cxId}, cxUserId ${cxUserId}`, err);
+    }),
+  ]);
+};
+
```

Comment:
Fine for now, but note that this will get spammy once user volume increases. Can revisit when we implement a message queue

Reply:
Those should be replaced by something like `Sentry.captureException(e);`, I'll add a TODO with the link to the issue and we can map those like this until implementation.